.github/workflows/build-ubuntu20.04-EXPERIMENTAL.yml

```diff
@@ -33,7 +34,7 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-20.04
+          - ubuntu-22.04
```

Comment:
Nice.  The file itself is still called build-ubuntu20.04-*.   Should that be adjusted?

Reply:
✔️